cogs/mod.py

```diff
@@ -210,6 +215,16 @@ def is_fast_join(self, member):
             self.fast_joiners[member.id] = True
         return is_fast
 
+    def is_mention_spam(self, message, config, current):
+        if not config.mention_count:
+            return False
+        mention_threshold = config.mention_count * 2
+        if self.by_mentions is None or self.by_mentions.rate != mention_threshold:
```

Comment:
I planned to add the `rate` property to `CooldownMapping` but forgot, so this would fail.